### PR TITLE
Add options to reject related to silently skip broken links

### PR DESCRIPTION
**Pull request by pezra**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,9 +2,8 @@ sudo: false
 cache: bundler
 language: ruby
 rvm:
-  - "1.9.3"
   - "2.0.0"
   - "2.1.0"
   - "2.2.0"
-  - jruby-19mode
-  - jruby-21mode
```
Comment (cschneid) [inline .travis.yml]: ya, screw you jruby!

**Comment by cschneid** [comment]
So the existing code let you fetch referenced links, do an operation on them?  And this patch is to handle what happens when the related link breaks?

File: .travis.yml (hunk 1 above)
Comment (pezra) [inline .travis.yml]: They screwed themselves by not supporting even the good parts of ruby 2.x. :smile:

**Comment by pezra** [comment]
In the old code you'd have to explicitly wrap you test block in a rescue if you wanted to just skip broken links. This this code you can do `.reject_related("author", ignore: :broken_links){ |repr| ...}` and it will just skip over (eg, leave them in the updated document) any links that rails a client error exception

**Pull request merged**